R/RLearner_fcregr_Arima.R

```diff
@@ -0,0 +1,85 @@
+#'@export
+makeRLearner.fcregr.Arima = function() {
+  makeRLearnerForecastRegr(
+    cl = "fcregr.Arima",
+    package = "forecast",
+    par.set = makeParamSet(
+      makeIntegerVectorLearnerParam(id = "order", len = 3L, lower = 0L, upper = Inf, default = c(0L, 0L, 0L)),
+      makeIntegerVectorLearnerParam(id = "seasonal", len = 3L, lower = 0L, upper = Inf, default = c(0L, 0L, 0L)),
+      makeLogicalLearnerParam(id = "include.mean", default = TRUE),
+      makeLogicalLearnerParam(id = "include.drift", default = FALSE),
+      makeNumericLearnerParam(id = "lambda", default = NULL, special.vals = list(NULL), when = "both"),
```

Comment:
I believe lambda for a box cox should have bounds of +- infinity, otherwise it would just be very large numbers like +- 1E15